py-shared/pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "quilt-shared"
+description = "Shared code for Quilt backend and lambdas"
+version = "0.1.0"
+requires-python = ">= 3.8, < 4"
+dependencies = [
+]
+
+[project.optional-dependencies]
+pydantic = [
+  "pydantic ~= 1.10",
+  "typing-extensions ~= 4.9",
+]
+boto = [
+  "boto3-stubs[s3] ~= 1.34",
+  "boto3 ~= 1.34",
+  "types-aiobotocore[s3] ~= 2.11",
+]
```

Comment:
do we need these in production?

Reply:
well... no. these extra groups are still open to change, it's just the first approximation